webui/react/src/components/LogViewer.module.scss

```diff
@@ -17,6 +17,7 @@
     font-family: 'Source Code Pro', monospace;
     font-size: var(--theme-sizes-font-small);
     height: 100%;
+    min-height: 500px;
```

Comment:
go ahead and land this, the fix for this is a bit more complicated (need to apply flex box all the way through to the logviewer from section). We'll address that separately.